### PR TITLE
Configurable window positioning behavior after matching image size

### PR DESCRIPTION
**Pull request by jdpurcell**
Implements #487.

### CODE REVIEW

File: src/qvoptionsdialog.cpp [hunk 1]
```diff
@@ -64,6 +64,7 @@ QVOptionsDialog::QVOptionsDialog(QWidget *parent) :
 #endif
 
     syncSettings(false, true);
+    connect(ui->windowResizeComboBox, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &QVOptionsDialog::windowResizeComboBoxCurrentIndexChanged);
```
Comment (jdpurcell) [inline src/qvoptionsdialog.cpp:67]: Fixed a bug in the existing code where this never got connected up.

**Comment by jurplel** [comment]
My main concern with this is the UX. It's a pretty confusing option. More direct language might help--but after revisiting this issue I am questioning if anyone is actually going to use this option. Unfortunately I don't remember my original motivation for the issue, but maybe this needs a rethink.

If we have an options cleanup with more tabs and stuff, this option should/could certainly exist, but right now the settings dialog is getting cluttered.

**Comment by jdpurcell** [comment]
@jurplel No worries, if you want we could close this for now and I could make a new pull request if you like the part with the `windowResizeComboBox` code deduplication and `currentIndexChanged` wire-up.

**Comment by jurplel** [comment]
Yes, I think that'd be great for now. Thanks!

**Pull request closed without merging**